MIGRATING.md

```diff
@@ -4,6 +4,88 @@ This guide explains what is needed to upgrade contracts when migrating over
 major releases of `cosmwasm`. Note that you can also view the
 [complete CHANGELOG](./CHANGELOG.md) to understand the differences.
 
+## 1.0.0 -> ?
+
+- There are changes to how we generate schemas, resulting in less boilerplace
+  maintenance for smart contract devs. The changes are backwards-compatible -
+  old contracts will continue working for a while, but it's highly recommended
+  to migrate now.
```

Comment:
Compared to 1.0, we generate a different output. It's all in one file now instead of many files. So I think it's safe to say the format changes.